.pipelines/pipeline.yaml

```diff
@@ -485,15 +487,12 @@ stages:
                 name: swiftv2_e2e
                 clusterName: "mtacluster"
                 region: $(REGION_SWIFTV2_CLUSTER_TEST)
-              swiftv2_dummy_e2e:
-                name: swiftv2_dummy_e2e
-                clusterName: "swiftv2dummy"
-                region: $(REGION_SWIFTV2_CLUSTER_TEST)
           steps:
-            - template: templates/delete-cluster.yaml
+            - template: templates/delete-cluster-swiftv2.yaml
               parameters:
                 name: $(name)
                 clusterName: $(clusterName)-$(commitID)
                 region: $(region)
                 sub: $(SUB_AZURE_NETWORK_AGENT_TEST)
                 svcConn: $(ACN_TEST_SERVICE_CONNECTION)
+                scenario: westcentralus
```

Comment:
scenario is westcentralus, can you explain more about this flag meaning?

Reply:
yes, it maps to a region/scenario configuration file that we use for swiftv2 runners